ui/app/routes/vault/cluster/settings/auth/configure.js

```diff
@@ -15,4 +17,12 @@ export default Route.extend({
       return this.store.peekRecord('auth-method', method);
     });
   },
+
+  setupController(controller) {
+    this._super(...arguments);
+    const methodData = findAuthMethod(controller.model.type);
+    // right now token is the only method that's not mountable
```

Comment:
this was actually a little bug, the title should say "Configure token" but because the helper failed it was just blank. Users likely aren't configuring methods in the UI, but nice to fix regardless
<img width="596" alt="Screenshot 2025-03-19 at 4 33 56 PM" src="https://github.com/user-attachments/assets/5d5c4b74-e769-4582-b390-cdf1af35c1ed" />